src/validator.js

```diff
@@ -412,6 +412,29 @@ export default class Validator
     }
 
     /**
+     * Append another validation to an existing field.
+     *
+     * @param  {string} name The field name.
+     * @param  {string} checks validations expression.
+     */
+    append(name, checks, options = {}) {
+        // No such field
+        if (!this.$fields[name]) {
+            return;
+        }
+
+        const checksArray = checks.split('|').map(rule => {
+            const normalizedRule = this._normalizeRule(rule, this.$fields[name].validations);
+            if (! normalizedRule.name) {
+                return;
```

Comment:
Thanks for the feedback! I will have a look in about 9 hours and create a fix for this.

I agree, keeping arrays nice and clean is always good! :)